raiden/tests/unit/test_cli.py

```diff
@@ -32,6 +43,50 @@ def test_cli_version(cli_runner):
     assert result_expected_keys == result_json.keys()
     assert result.exit_code == 0
 
+    result = cli_runner(run, ["--version"])
+    assert "Hint: Use '" in result.output and "version' instead" in result.output
+    assert result.exit_code == 0
+
+
+def mock_raises(exception):
+    def f(*_, **__):
+        raise exception
+
+    return f
+
+
+def test_run_error_reporting(cli_runner, monkeypatch):
+    result = cli_runner(run, ["--transport", "udp"])
+    assert "Invalid value" in result.output and "--transport" in result.output
```

Comment:
Relying on the error message is not a good idea. The failure of this test is not useful.

Reply:
The `cli_runner` could also fail because of some missing option or wrong config. Making sure it failed with a transport option related error message is more reliable.